TileFormats/FeatureTable/README.md

```diff
@@ -70,4 +70,4 @@ In JavaScript, a `TypedArray` cannot be created on data unless it is byte-aligne
 For example, a `Float32Array` must be stored in memory such that its data begins on a byte multiple of four since each `float` contains four bytes.
 
 The string generated from the JSON header should be padded with space characters in order to ensure that the binary body is byte-aligned.
-The binary body should also be padded if necessary when there is data following the Feature Table.
+The binary body should also be padded to a multiple of eight bytes. This ensures byte-alignment for the embedded glTF in Batched 3D Models and Instanced 3D Models.
```

Comment:
Are we sure about this?  I thought the guarantees would be:
* Typed arrays in the Feature Table's binary body are byte-aligned to the size of their component type, e.g., 4 bytes for 
`float`.
* Typed arrays in the Batch Table's binary body are byte-aligned to the size of their component type, e.g., 4 bytes for `float`.
* In `b3dm` and `i3dm` (put the wording in their specs), the Binary glTF is 8-byte aligned, so glTF's byte-alignment guarantees can be meet with respect to the tile's entire arraybuffer.